src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write your code here
+const logoEl = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve) => {
+  const messageEl = document.createElement('div');
+
+  messageEl.classList.add('message');
+  messageEl.textContent = 'Promise was resolved!';
+  logoEl.onclick = () => resolve(messageEl);
+});
+
+promise1.then((messageEl) => document.body.append(messageEl));
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    const errorEl = document.createElement('div');
+
+    errorEl.classList.add('message', 'error-message');
+    errorEl.textContent = 'Promise was rejected!';
+    reject(errorEl);
+  }, 3000);
+});
+
+promise2.catch((errorEl) => document.body.append(errorEl));
```

Comment:
The task requires adding both success and error handlers for the promises. While the error handler for `promise2` is implemented using `catch`, it would be more consistent with the task requirements to also include a `then` handler for `promise2` to handle any potential success case, even if it's not expected to resolve successfully. This ensures that both success and error handlers are explicitly defined for each promise.